Description: template for firmware for Spyder2 device
 Contains template for argyll-firmware-spyder2 package that may be created
 by end user to distribute firmware for Spyder2 to multiple Debian systems
Author: Roland Mas <lolando@debian.org>
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/control
@@ -0,0 +1,17 @@
+Source: argyll-firmware-spyder2
+Section: non-free/graphics
+Priority: optional
+Maintainer: Roland Mas <lolando@debian.org>
+Build-Depends: debhelper (>= 7)
+Standards-Version: 3.9.1
+
+Package: argyll-firmware-spyder2
+Architecture: all
+Recommends: argyll
+Depends: ${misc:Depends}
+Description: ColorVision Spyder2 firmware for Argyll
+ Argyll is an open source, ICC compatible color management system.  It
+ can drive display calibrators such as the ColorVision Spyder2, which
+ requires a firmware to be uploaded to the device.  This package is
+ generated from the drivers CD provided with the Spyder2, and contains
+ the firmware ready to be used by Argyll.
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/rules
@@ -0,0 +1,10 @@
+#! /usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	cp /usr/local/share/color/spyd2PLD.bin .
+
+override_dh_auto_clean:
+	rm -f spyd2PLD.bin
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/changelog
@@ -0,0 +1,12 @@
+argyll-firmware-spyder2 (1.1) unstable; urgency=low
+
+  * Moved firmware to /usr/share/color, which is freedesktop.org-compliant
+    and (more to the point) where Argyll 1.2.0 looks for it.
+
+ -- Roland Mas <lolando@debian.org>  Thu, 05 Aug 2010 15:32:53 +0200
+
+argyll-firmware-spyder2 (1.0) unstable; urgency=low
+
+  * Package automatically generated from the Spyder2 driver CD.
+
+ -- Roland Mas <lolando@debian.org>  Fri, 17 Apr 2009 22:28:37 +0200
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/compat
@@ -0,0 +1 @@
+7
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/copyright
@@ -0,0 +1,8 @@
+The binary package contains the proprietary firmware for the
+ColorVision Spyder2 device.  As such, it is not meant to be
+distributed.
+
+The source package itself (which builds the binary package) is
+Copyright 2009, Roland Mas, and can be distributed under the terms of
+the GNU General Public License, version 2 or (at your option) any
+later version published by the Free Software Foundation.
--- /dev/null
+++ b/firmware-package-builder/argyll-firmware-spyder2/debian/install
@@ -0,0 +1 @@
+spyd2PLD.bin usr/share/color
